src/pages/orgs/Overview.tsx

```diff
@@ -203,9 +239,43 @@ const Overview = () => {
                             variant="contained"
                             onClick={handleInteract}
                             disabled={disabled || attemptingInteract}
+                            sx={{ backgroundColor: "rgba(248, 19, 19, 0.88)" }}
```

Comment:
This makes the button red for all interactions (e.g. joining); instead it should check the value of `interactString` (I will handle this)